_chapters/chapter_01.md

```diff
@@ -9,22 +9,30 @@ layout: rails_tutorial
 {% endsectionheader %}
 
 {% protip %}
-  Throughout the tutorial, you're going to run a lot of Terminal commands. For example, we might ask you to do something like
+  Throughout the tutorial, you're going to run a lot of commands on the command line. For example, we might ask you to do something like
 
-  $ Run `pwd` to print your current working directory.
+  > Run `$pwd` to print your current working directory.
 
-  To run the command, you'll need to type it in Terminal and hit `enter` to execute it.
+  The place that you run these commands will be different based on the type of computer you have: 
+
+  * If you're using a mac, you'll type the command into *Terminal* and hit `enter` to execute it.
+
+  * To run this command on Windows you'll need to type it into the *Command Prompt* then press `enter`. 
+  
+  Throughout this tutorial we'll be referring to both Terminal and Command Prompt as "Terminal". 
 {% endprotip%}
 
 {% steps %}
 {% list %}
-  1.  Open Terminal.
+  1.  Get started by opening your Terminal.
```

Comment:
I've been writing "the" Terminal instead of "your" Terminal throughout the tutorial. I *think* I like "the Terminal" better, but I can be convinced otherwise.